adage/analyze/api.py

```diff
@@ -5,7 +5,7 @@
 from organisms.api import OrganismResource
 from genes.api import GeneResource
 from tastypie import fields, http
-from tastypie.resources import Resource, ModelResource
+from tastypie.resources import Resource, ModelResource, convert_post_to_VERB
```

Comment:
True. If you can create a test case whose POST request is longer than 4KB and confirm that data is retrieved from database successfully, it will be good enough. Thanks.